### ISSUE TITLE
API needs pagination

### ISSUE THREAD

**Issue by vsoch** [poldracklab/open_fmri#25]
There was a post on neurostars about the API, and I checked it out, and it seemed to load pretty slowly. I did a simple test:

```
  import time
  import requests
  import numpy

  times = []

  for i in range(100):
      print "loop %s of 100" %(i)
      start = time.time() 
      response = requests.get('https://openfmri.org/dataset/api/')
      end = time.time()    
      times.append(end-start)

  print "Average time is %s" %(numpy.mean(times))
  Average time is 5.94425868034
```

and it takes almost 6 seconds per load! I think this would greatly benefit with the addition of pagination!

**Comment by rwblair** [comment]
Good idea, thats pretty slow. I'll give pagination a try since its built into django rest framework. Looks like some one has made an add on that can cache api responses that I might add if pagination alone isn't good enough.
https://chibisov.github.io/drf-extensions/docs/#caching
